README.md

```diff
@@ -1 +1,13 @@
-# parse-big
\ No newline at end of file
+# beyond-global-json
+
+Javascript uses floating numbers to store all numbers. That means that past a certain point (Number.MAX_SAFE_INTEGER), integers will be incorrect due to floating point precision.
+
+beyond-global-json module provides a simple overwrite of the following global JSON object methods, to handle the described above:
+* `parse`
+* `stringify`
+
+### Usage
+
+```javascript
+require( 'beyond-global-json' );
```

Comment:
@avinoamr It's already invoking the `overwrite` method when being required:
`module.exports = overwrite()`